src/index.js

```diff
@@ -14,6 +14,9 @@ const MINIMAL_WORKER_TIME        = 30000;
 const TESTCAFE_CLOSING_TIMEOUT   = 10000;
 const TOO_SMALL_TIME_FOR_WAITING = MINIMAL_WORKER_TIME - TESTCAFE_CLOSING_TIMEOUT;
 
+const AUTH_FAILED_ERROR = 'Authentication failed. Please set the BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY ' +
+                          'environment variables to the correct pair of an username and an access key.';
```

Comment:
`Authentication failed. Please assign the correct username and access key to the BROWSERSTACK_USERNAME and BROWSERSTACK_ACCESS_KEY environment variables.`